src/renderer/components/dashboard/Send.vue

```diff
@@ -253,10 +250,6 @@ export default {
     currency() {
       this.cleanAmount();
     },
-
-    amount() {
-      this.cleanAmount();
```

Comment:
Remove from 'watchers'.

Reply:
Have you tried debounce?

```
watch: {
    ...,
    amount: _.debounce(this.cleanAmount, 200), // time is in milliseconds... 200 is a shot in the dark
},

Reply:
Sure - but that still would delete after 200 MS.

Reply:
If I type '1.000' then 200 MS go by - it still will revert to 1

Reply:
Since the value is being watched with every keystroke - even if you have a delay/debounce - the problem still exists where it will alter the value incorrectly.

Reply:
https://lodash.com/docs/4.17.10#debounce

Reply:
The problem is if you are trying to type 1.0001 for example; right now it keeps deleting your 0's before you can get to the point of typing the 1. The idea with this is to not clean up the entered value until they navigate away or click send.

Reply:
I'm not sure it is a great idea to be cleaning up their number while they still might be typing it though; consider if they are copying an amount they need to send from a paper.
Say they are sending 0.40000327 of a valuable asset, and they pause a to look back at the paper after typing 0.40000; then it cleans it to 0.4 and they type 327, leaving them with the number 0.4327...
I guess that is what the confirmation dialog is about though; so as long as we use a delay greater than 1 second, I think I'm ok with cleaning the amount using the debounce and with blur action.

Reply:
Yeah, let's forget it. Just an idea, wasn't sure if it would work properly.